cartridge/lua-api/boxinfo.lua

```diff
@@ -108,9 +108,15 @@ local function get_info(uri)
             vars.http_port = srv_name.port
         end
 
+        local handle = require('io').popen("git describe --long --abbrev=7 --tags")
```

Comment:
Don't use `popen` and `git describe`. The original issue  (https://github.com/tarantool/cartridge-cli/issues/528) is about `require'VERSION'`. You should also cover the case when VERSION.lua is absent